--- dbm-camldbm-1.3/Makefile.orig	2022-01-29 16:02:30.175985113 +0100
+++ dbm-camldbm-1.3/Makefile	2022-01-29 16:06:16.238093763 +0100
@@ -57,13 +57,13 @@ depend:
 	$(OCAMLDEP) *.ml *.mli > .depend
 
 install::
-	if test -f dllcamldbm.$(SO); then mkdir $(STUBLIBDIR) || echo Ok; cp dllcamldbm.$(SO) $(STUBLIBDIR)/; fi 
-	cp libcamldbm.$(A) $(LIBDIR)/
-	cd $(LIBDIR) && ranlib libcamldbm.$(A)
-	cp dbm.cmx dbm.cma dbm.cmxa dbm.cmi dbm.mli $(LIBDIR)/
-	cp dbm.$(A) $(LIBDIR)/
-	cd $(LIBDIR) && ranlib dbm.$(A)
-	if test -f dbm.cmxs; then cp dbm.cmxs $(LIBDIR)/; fi
+	if test -f dllcamldbm.$(SO); then mkdir $(DESTDIR)$(STUBLIBDIR) || echo Ok; cp -p dllcamldbm.$(SO) $(DESTDIR)$(STUBLIBDIR)/; fi 
+	cp -p libcamldbm.$(A) $(DESTDIR)$(LIBDIR)/
+	cd $(DESTDIR)$(LIBDIR) && ranlib libcamldbm.$(A)
+	cp -p dbm.cmx dbm.cma dbm.cmxa dbm.cmi dbm.mli $(DESTDIR)$(LIBDIR)/
+	cp -p dbm.$(A) $(DESTDIR)$(LIBDIR)/
+	cd $(DESTDIR)$(LIBDIR) && ranlib dbm.$(A)
+	if test -f dbm.cmxs; then cp -p dbm.cmxs $(DESTDIR)$(LIBDIR)/; fi
 
 clean::
 	rm -f *.cm* *.$(O) *.$(A) *.$(SO)
